code/__DEFINES/atmospherics/atmos_piping.dm

```diff
@@ -39,8 +39,8 @@
 #define TANK_RUPTURE_PRESSURE (35.*ONE_ATMOSPHERE)
 /// Boom 3x3 base explosion
 #define TANK_FRAGMENT_PRESSURE (40.*ONE_ATMOSPHERE)
-/// +1 for each SCALE kPa aboe threshold
-#define TANK_FRAGMENT_SCALE (6.*ONE_ATMOSPHERE)
+/// Range scaling constant for tank explosions. (Higher means tank explosions scale slower)
+#define TANK_FRAGMENT_SCALE (42.*ONE_ATMOSPHERE)
```

Comment:
I assume it's a convention in other languages to ensure that the number is forced to be a float. I really just included it because the original constant included it and for parity/unity/symmetry/whatever with the other constants.

Reply:
It's older then r4407. Let's just call it soul and go on about our day

Reply:
God damn github not letting me open a new conversation.
You're centering the transition from the old equation to the new one at 70. The volume of a plasma or o2 tank. But doesn't a ttv merge its contents, including the volume, on explosion?

Reply:
It was intended to have the same maxcap pressure at 140L but I appear to have messed up with the math. One sec.

Reply:
Yeah. I must have missed the hardcoded x2 in dyn_explosion or mistakenly balanced around 70L tanks. I should be able to fix it in 3 minutes once my laptop boots.

Reply:
Done.